recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,17 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+SRCREV_kernel="${AUTOREV}"
+
+SRC_URI = "git://github.com/xin3liang/linux;branch=hikey-tracking-integration-devel-drm;name=kernel \
+           file://defconfig  \
```

Comment:
you didn't change defconfig, though you are moving the kernel from 3.18 to 4.5. this is suspicious. is that really what you want?